Tests/FluentAssertions.Specs/Execution/FallbackTestFrameworkTests.cs

```diff
@@ -0,0 +1,24 @@
+using FluentAssertions.Execution;
+using Xunit;
+
+namespace FluentAssertions.Specs.Execution;
+
+public class FallbackTestFrameworkTests
+{
+    [Fact]
+    public void When_accessing_fallback_test_framework_it_is_always_available()
```

Comment:
We're trying to migrate (at least new test classes) to an in our opinion less noisy naming schema.
So instead of the sometimes inflexible `when_foo_then_bars` format, this method could e.g. be named `The_fallback_test_framework_is_available()`.

See https://enterprisecraftsmanship.com/posts/you-naming-tests-wrong/

Reply:
For me it gets more and more difficult to name the tests according your thoughts...
Is your version above a general suggestion or a requirement?
According to the linked article it is correct, I guess.

Reply:
done